index.html

```diff
@@ -2,20 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
 <!-- inserted chosen css framework -->
-<<<<<<< HEAD
```

Comment:
TIL this is a merge conflict marker. We'll need fix/remove these when we encounter them.